test/test_nn.py

```diff
@@ -16193,6 +16193,53 @@ def helper(n, c, h, w, output_height, output_width, contig):
             helper(4, 8, 9, 14, 5, 8, contig)
             helper(4, 8, 11, 11, 1, 1, contig)
 
+    @dtypes(torch.float, torch.double)
+    def test_pooling_max_nhwc(self, dtype):
```

Comment:
tests here automatically get `device` argument, and it's populated with correct device
```suggestion
    def test_pooling_max_nhwc(self, device, dtype):
```